src_tests/test_quantum_computations_utilities.py

```diff
@@ -28,9 +26,9 @@ def test_unitarity_of_matrices_generated_by_haar_random_unitary_method(self):
     @staticmethod
     def __complex_almost_equal(a: complex128, b: complex128) -> bool:
         difference = a - b
-        return abs(difference) < float_info.epsilon
+        return abs(difference) < 1e-13
```

Comment:
Have you considered using https://numpy.org/doc/stable/reference/generated/numpy.finfo.html ? It seems that `numpy.finfo(numpy.complex128).eps` is what you're looking for.

Reply:
Actually, with `allclose` method, which works on complex, I don't even need this method anymore. I can hardly imagine why I didn't check numpy for complex numbers closeness methods. Thanks.